### DISCUSSION TITLE
How to use Rendering

### DISCUSSION THREAD

**Discussion by echonok** [shakiba/planck.js#172] [Q&A] [answered]
Hi! You made a brilliant job.

I can't understand how to use rendering. In the last part of Wiki you create render(), but don't use it. How to start animation without Testbed?

**Comment by Toby222** [comment, marked as answer]
https://github.com/RealPeha/planck-renderer
This, while not really great or complex, shows a simple implementation of a planck renderer
renderer-run is a Runner responsible for ticking the physics, renderer-canvas is responsible for actually drawing the world

**Reply by echonok** [reply]
Yeah, thanks!
It turned out that this renderer has problems with building SSR. Managed to find a solution by using planck-renderer as js-bundle and requiring it, not importing
